backend/app/utils/config_definitions/validations.py

```
@@ -43,7 +68,7 @@ def validate_schema_structure(json_schema: Dict[str, Any]) -> None:
             return
         jsonschema.Draft7Validator.check_schema(json_schema)
     except jsonschema.exceptions.SchemaError as e:
-        raise ValueError(f"Invalid JSON Schema provided: {e.message}")
+        raise validation_error("json_schema", e.message.split("\n", 1)[0])
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)